internal/serve/httphandler/verifiy_receiver_registration_handler_test.go

```diff
@@ -693,20 +689,16 @@ func Test_VerifyReceiverRegistrationHandler_VerifyReceiverRegistration(t *testin
 		handler.ReCAPTCHAValidator = reCAPTCHAValidator
 		reCAPTCHAValidator.On("IsTokenValid", mock.Anything, "token").Return(true, nil).Once()
 
-		apTxPatches := []anchorplatform.APSep24Transaction{{
-			ID:                 "test-transaction-id",
-			Status:             "pending_anchor",
-			SEP:                "24",
-			Kind:               "deposit",
-			DestinationAccount: validClaims.SEP10StellarAccount(),
-			Memo:               validClaims.SEP10StellarMemo(),
-			KYCVerified:        true,
-		}}
+		apTxPatche := anchorplatform.APSep24TransactionPatchPostRegistration{
```

Comment:
Perhaps `apTxPatch`?

Reply:
Addressed in https://github.com/stellar/stellar-disbursement-platform-backend/pull/40/commits/f6a244b2655f8926423b33cfc0d0efaefba13996